cluster/cluster.go

```diff
@@ -17,7 +17,7 @@ type Cluster interface {
 	SomeControllers([]flux.ResourceID) ([]Controller, error)
 	Ping() error
 	Export() ([]byte, error)
-	Sync(SyncDef) error
+	Sync(SyncDef, bool) error
```

Comment:
Alternate suggestion: accept the names of things that are expected to cause problems, i.e., whatever had problems last time.